apps/editors/templates/editors/review.html

```diff
@@ -163,25 +163,20 @@ <h3 id="history">
       </div>
 
       <div class="review-actions-section review-actions-files data-toggle"
-           data-value="{{ actions_minimal|join("|") }}"{% if allow_unchecking_files %} data-uncheckable="1"{% endif %}>
-        <label for="id_addon_files"><strong>{{ form.addon_files.label }}</strong></label>
+           data-value="{{ actions_minimal|join("|") }}">
+        <label for="id_addon_files"><strong>{{ _('Files:') }}</strong></label>
         <ul>
-            {% for pk, label in form.fields.get('addon_files').choices %}
+            {% for file in form.helper.version.unreviewed_files %}
```

Comment:
This knows a lot about these objects. Finding the files seems like a good use case for the helper and in my opinion the form should be wrapping that but I think we access the helper elsewhere.

```jinja
{% for file in form.unreviewed_files %}
{# or #}
{% for file in form.helper.unreviewed_files %}